packages/contracts-bedrock/src/universal/SafeSend.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
```

Comment:
This should be `^0.8.0` so it can be imported by any contract, its more of a library